articles/tech-resources/a-b-testing-sparkpost.md

```diff
@@ -5,39 +5,42 @@ description: "A B Testing in Spark Post An A B test in summary is a common metho
 
 ### A/B Testing in SparkPost 
 
-An A/B test, in summary, is a common method of comparing templates to see which one performs better. SparkPost provides the means to create new A/B tests and view completed results through our A/B Testing API.
+A/B testing is a common method of comparing templates to see which one performs better. SparkPost provides the means to create A/B tests and view completed results through our A/B Testing API.
 
 **What benefits do I get from using A/B testing?** 
 
-An A/B test is a method of comparing 2 or more templates to see which one performs better. That is, which one provides a higher unique open or click rate. The SparkPost A/B Testing API allows users to compare multiple templates in the context of transactional or app generated messages, rather than lists, to see which one performs better for a set number of recipients (Sample Size) or over a set period of time. 
+A/B testing is a means of determining what content drives more engagement (greater open/click rates). 
 
-Some studies have shown that doing A/B testing on email content (vs. not doing any testing) can lift open rates by 13% or more. So here's a step by step guide to A/B Testing at SparkPost to get your started. Note that enhancements are coming over the months ahead (we're agile too!) - so please give us feedback on what you would like to see improved in our [Beta Program](https://www.sparkpost.com/sparkpost-beta-tester/).
+The SparkPost A/B Testing API allows users to compare multiple templates in the context of standard marketing campaigns and transactional/app-generated messages to see which one performs better for a set number of recipients (sample size) or over a set period of time. 
```

Comment:
It's a small thing, but I want to de-emphasize marketing campaigns because those generally are done as lists. What differentiates what we've built is that it is for the  one-off use case, which is typically transactional or app-generated. While the Support article isn't a piece of marketing collateral, it's what I have right now so want to emphasize that point.

Reply:
Also just noticed you removed the reference to the Beta program -- please keep that in for now. We're move it once the feature (and our nesceant) beta program are more mature.